src/h5web/App.tsx

```diff
@@ -51,51 +48,21 @@ function App(): JSX.Element {
           flex={isExplorerOpen ? 0.75 : 1}
           minSize={500}
         >
-          <div className={styles.toolbar}>
```

Comment:
The explorer and role toggler were moved in a dedicated component `BreadcrumbsBar` that also implements the display of the breadcrumbs